atc/atccmd/command.go

```diff
@@ -239,6 +239,8 @@ type RunCommand struct {
 	SystemClaimKey        string   `long:"system-claim-key" default:"aud" description:"The token claim key to use when matching system-claim-values"`
 	SystemClaimValues     []string `long:"system-claim-value" default:"concourse-worker" description:"Configure which token requests should be considered 'system' requests."`
 	EnableArchivePipeline bool     `long:"enable-archive-pipeline" description:"Enable /api/v1/teams/{team}/pipelines/{pipeline}/archive endpoint."`
+
+	EnableBuildRerunWhenWorkerDisappear bool `long:"enable-rerun-when-worker-disappear" description:"Enable automatically build rerun when worker disappears"`
```

Comment:
Small critique - I would add an "s" at the end `EnableBuildRerunWhenWorkerDisappears`

Reply:
fixed.